druid/src/core.rs

```diff
@@ -641,10 +645,9 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                         // Only send FocusChanged in case there's actual change
                         if old != new {
                             self.state.has_focus = change;
-                            let event = LifeCycle::FocusChanged(change);
-                            self.inner.lifecycle(ctx, &event, data, env);
+                            substitute_event = Some(LifeCycle::FocusChanged(change));
                         }
-                        false
+                        true
```

Comment:
The `true` should be inside the `if old != new` block and `false` otherwise. An `else` block with `false` would work.

Reply:
good catch